spring-kafka/src/main/java/org/springframework/kafka/core/StreamsBuilderFactoryBean.java

```diff
@@ -62,13 +63,23 @@
 	private volatile boolean running;
 
 	public StreamsBuilderFactoryBean(StreamsConfig streamsConfig) {
+		this(streamsConfig, new CleanupConfig());
+	}
+
+	public StreamsBuilderFactoryBean(StreamsConfig streamsConfig, CleanupConfig cleanupConfig) {
```

Comment:
Needs Javadoc and `@since 2.1.2`; we should add javadocs to the other ctors too.

Reply:
I can leave this comments to you, or if would you suggest something I can add it.